api/src/main/java/com/microsoft/gctoolkit/GCToolKit.java

```diff
@@ -30,13 +35,40 @@
      * @param logFile GCLogFile DataSource
      * @return JavaVirtualMachine implementation.
      */
-    private static JavaVirtualMachine loadJavaVirtualMachine(GCLogFile logFile) {
+    private JavaVirtualMachine loadJavaVirtualMachine(GCLogFile logFile) {
         return ServiceLoader.load(JavaVirtualMachine.class)
                 .stream()
                 .map(ServiceLoader.Provider::get)
                 .filter(jvm -> jvm.accepts(logFile))
                 .findFirst()
-                .orElseThrow(() -> new ServiceConfigurationError("No suitable service provider found"));
+                .orElseThrow(() -> new ServiceConfigurationError("No suitable JavaVirtualMachine implementation found"));
+    }
+
+    private DataSourceBus setupDataSourceBus() {
```

Comment:
Spoke to Kirk about these 3 methods, it's part of a refactoring to make Vertx an impl of a bus as opposed to being the bus.  Makes sense so I'm OK with this here (as long as we raise an issue to complete the refacror)